cvat-core/src/index.ts

```diff
@@ -150,6 +152,19 @@ export default interface CVATCore {
     frames: {
         getMeta: any;
     };
+    requests: {
+        list: () => Promise<{ requests: Request[], count: number }>;
+        listen: (
+            rqID: string | null,
+            options?: {
+                callback?: (request: Request) => void,
```

Comment:
Shall this callback be optional? What is a reason to listen for something if we do not want to get updates?

Reply:
Why not, if some code wants to listen but dont need to update any UI

Reply:
Could you please provide example?